packages/headless/src/features/did-you-mean/did-you-mean-actions.ts

```diff
@@ -20,3 +20,11 @@ export const applyDidYouMeanCorrection = createAction(
   'didYouMean/correction',
   (payload: string) => validatePayload(payload, requiredNonEmptyString)
 );
+
+export const enableFallbackSearchOnEmptyQueryResults = createAction(
```

Comment:
If we use that name `fallbackSearchOnEmptyQueryResults` and we eventually change the name in our API, are we gonna be able to change the name in the headless state or it should remain the same?

Reply:
> If we use that name fallbackSearchOnEmptyQueryResults and we eventually change the name in our API, are we gonna be able to change the name in the headless state or it should remain the same?

We prefer to stick to the Search API name. Now is the time to change it on your end, or "wait for a major version" kinda thing.

I will wait to merge/publish this until it's decided.

Reply:
The headless option. Which means there's some divergence from Search API, which is not ideal. 

The problem is related to documentation/glossary. 

ie: if someone searches `automaticQueryCorrection` they might not find Search API doc for it, or vice versa if they search for `enableFallbackSearchOnEmptyQueryResults`.

But, when we can offer a "better" abstraction over what the search API parameter actually, it's worth it.